Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Core.cpp

```diff
@@ -287,20 +299,20 @@ namespace ScriptCanvasEditor
         if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
         {
             serializeContext->Class<SourceHandle>()
-                ->Version(0)
+                ->Version(1)
                 ->Field("id", &SourceHandle::m_id)
-                ->Field("path", &SourceHandle::m_path)
+                ->Field("path", &SourceHandle::m_relativePath)
```

Comment:
if we prefer to use relative path and serialize relative path only, do we still need to store m_path?

Reply:
Yeah, I only want the conversion between relative and absolute done once.